src/openms/include/OpenMS/FILTERING/TRANSFORMERS/SpectraMerger.h

```diff
@@ -297,22 +298,77 @@ namespace OpenMS
       exp.sortSpectra();
     }
 
+    /**
+     * @brief check if the first and second mzs might be from the same mass
+     *
+     * @param mz1 the first m/z value
+     * @param mz2 the second m/z value
+     * @param tol_ppm tolerance in ppm
+     * @param max_c maximum possible charge value
+     */
+    bool areMassesMatched(double mz1, double mz2, double tol_ppm, int max_c)
```

Comment:
this looks like a static function which could also go into MathFunctions?

But make it static in case it stays here

Reply:
Done